src/applications/claims-status/components/ClaimsListItemV3.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import { getClaimType } from '../utils/helpers';
+
+const statusMap = {
+  CLAIM_RECEIVED: 'Step 1 of 5: Claim received',
+  INITIAL_REVIEW: 'Step 2 of 5: Initial review',
+  EVIDENCE_GATHERING_REVIEW_DECISION:
+    'Step 3 of 5: Evidence gathering, review, and decision',
+  PREPARATION_FOR_NOTIFICATION: 'Step 4 of 5: Preparation for notification',
+  COMPLETE: 'Step 5 of 5: Closed',
+};
+
+function getStatusDescription(status) {
+  return statusMap[status];
+}
+
+const formatDate = date => moment(date).format('MMMM D, YYYY');
```

Comment:
I would swap`moment` for `date-fns` here

Reply:
I would take a look at the `buildDateFormatter` function in the helpers file

Reply:
This line would be `const formatDate = buildDateFormatter(DATE_FORMATS.LONG_DATE);`